tests/dspa_v2_test.go

```diff
@@ -29,7 +29,16 @@ import (
 
 var _ = Describe("A successfully deployed DSPA", func() {
 
-	podCount := 8
+	podCount := 6
+
+	var isTestDspa bool
+
+	BeforeEach(func() {
+		isTestDspa = DSPA.Name == "test-dspa"
+		if isTestDspa {
+			podCount = 8
+		}
+	})
```

Comment:
don't like this, we should not hard code a dspa name here

instead, detect whether external object store is being used, if so decrement podcount
do the same with external database